test/dag-node-test.js

```diff
@@ -29,10 +29,10 @@ module.exports = (repo) => {
         expect(Buffer.isBuffer(node.data)).to.be.true.mark()
         expect(node.size).to.be.above(0).mark()
 
-        util.serialize(node, (err, serialized) => {
+        DAGNode._serialize(node, (err, serialized) => {
```

Comment:
Take that back, I understand what you mean with the comment. However, the util API doesn't make sense to me. Why would they need to be namespaced under utils instead of haing them on the top level? Ie.
```javascript
const DAGNode = require('dag-node')
const node = DAGNode.create(...)
DAGNode.serialize(node, ...)
DAGNode.deserialize(node, ...)
// etc.
```